stdlib/public/Platform/POSIXError.swift

```diff
@@ -1,254 +1,466 @@
-// FIXME: Only defining POSIXErrorCode for Darwin at the moment.
```

Comment:
This entire file is not formatted correctly.  Could you please

- Use 2-space indentation
- Align top-level declarations without any leading whitespace.